demo-playground/src/main/java/com/google/android/flexbox/recyclerview/FlexItemAdapter.java

```diff
@@ -52,6 +55,8 @@ public FlexItemAdapter(AppCompatActivity activity, FlexboxLayoutManager layoutMa
     public FlexItemViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View view = LayoutInflater.from(parent.getContext())
                 .inflate(R.layout.viewholder_flex_item, parent, false);
+
+        Log.d(TAG, "onCreateViewHolder. ViewType: " + viewType + ", Inflated view: " + view);
```

Comment:
It's shown only in the demo app, but used only for the development purpose. Deleted.